src/repositories/rogue.js

```diff
@@ -594,11 +622,19 @@ export const getSignupsById = async (ids, options) => {
     ids,
   });
 
-  const idQuery = ids.join(',');
+  const queryString = {
+    filter: {
+      ids: ids.join(','),
```

Comment:
I think this needs to be `id` singular?
```suggestion
      id: ids.join(','),
```

Reply:
Yes, thank you!